cogs/audio.py

```diff
@@ -1491,26 +1491,26 @@ def _clean_url(self, url):
 
         # Checking already connected, will join if not
 
-        try:
-            self.has_connect_perm(author, server)
-        except AuthorNotConnected:
-            await self.bot.say("You must join a voice channel before I can"
-                               " play anything.")
-            return
-        except UnauthorizedConnect:
-            await self.bot.say("I don't have permissions to join your"
-                               " voice channel.")
-            return
-        except UnauthorizedSpeak:
-            await self.bot.say("I don't have permissions to speak in your"
-                               " voice channel.")
-            return
-        except ChannelUserLimit:
-            await self.bot.say("Your voice channel is full.")
-            return
-
         if not self.voice_connected(server):
```

Comment:
This is not the correct condition, or at least not without additional conditions, as the else condition prior existing that you left in place would also pass this condition.